static/bidder-params/jixie.json

```diff
@@ -0,0 +1,31 @@
+  {
+    "$schema": "http://json-schema.org/draft-04/schema#",
+    "title": "Jixie Adapter Params",
+    "description": "A schema which validates params accepted by the Jixie adapter",
+    "type": "object",
+    "properties": {
+      "unit" : {
+        "type": "string",
+        "description": "The unit code of an inventory target",
+        "minLength": 18
```

Comment:
> added minLen 18 to be our criteria for unit. didn't know there is this minLength Thanks!!

@jxdeveloper1 there's even a `"pattern"` restriction that validates that the value of a field matches a regular expression. Do you think it'd be useful in our context? 

For instance, if we wanted `unit` to always look like `"1000008-AA77BB88CC"`, we could probably:
```
 6     "properties": {
 7       "unit" : {
 8         "type": "string",
 9         "description": "The unit code of an inventory target",
10 -       "minLength": 18
   +       "pattern": "^[0-9]{7}-[0-9A-Z]{10}$"
11       },
static/bidder-params/jixie.json
```
But if you want to use `"minLenght"` I'm ok either way.

Reply:
Thanks. I think minLen 18 is a good enough rule.
Imposing more rules might create other problems. But I really appreciate the tip!

Reply:
Yes stupid me forgot to remove the processImp function. (And I was wondering why my coverage dropped from > 80 to ~ 70). It is removed.

Reply:
Much thanks for your patience. I have a question:
So if the request is really coming thru e.g. the openrtb2/auction call, then I know the check 
for "presence of unit property, 18-char min" is applied.
But I noticed 

1/This check is not applied on those validation-test cases jixietests/exemplary/blabla.json
-- that's because the validation test are meant to test the adaptor go-code whereas the 
"unit, length>=18" check is jurisdiction of the prebid-server; therefore that check is not exercised in the validation test.
Is that a correct understanding?

2/Tests jixietests/exemplary/*.json, supplementary/*.json do not involve really calling of our endpoint (reason as I guessed in 1/) but then jixietests/params/race/* actually does trigger the production endpoint it seem <-- What is this race stuff for anyways?